src/components/Users/Users.js

```diff
@@ -1,15 +1,86 @@
-import User from '../User/User';
-import './Users.css';
-
-const Users = ({ users = [] }) => {
-  return (
-    <article className="Users">
-      {users.map((user) => {
-        const { id } = user;
-        return <User key={id} user={user} />;
-      })}
-    </article>
-  );
+import { useEffect } from "react";
+import User from "../User/User";
+import "./Users.scss";
+
+const Users = ({
+  users = [],
+  searchInput,
+  expanded,
+  setExpanded,
+  loading,
+  error,
+}) => {
+  useEffect(() => {
+    const expandedList = window.localStorage.getItem("usersList");
+    if (expandedList !== null) {
+      setExpanded(JSON.parse(expandedList));
+    }
+  }, []); // eslint-disable-line
+
+  const handleToggleExpanded = (id) => {
+    if (!expanded.includes(id)) {
+      const newExpanded = [...expanded, id];
+      setExpanded(newExpanded);
+      window.localStorage.setItem("usersList", JSON.stringify(newExpanded));
```

Comment:
Well done here. I am glad to see that `newExpanded` is passed to both `setExpanded` and local storage.